index.bs

```diff
@@ -1457,6 +1459,79 @@ Resignation from a Group</h3>
 	See “Exclusion and Resignation from the Working Group” in the W3C Patent Policy [[PATENT-POLICY]]
 	for information about obligations remaining after resignation from certain groups.
 
+<h3 id="tooling">
+Tooling</h3>
+
+	For [=W3C Groups=] operating under this Process,
+	a core operating principle is to allow access across disabilities,
+	across country borders,
+	and across time.
+	Thus in order to allow all would-be participants to effectively participate,
+	to allow future participants and observers to understand the rationale and origins of current decisions,
+	and to guarantee long-lived access to its publications,
+	W3C requires that:
+
+	<ul>
+		<li>
+			All reports, publications, or other deliverables
+			produced by the group for public consumption
+			(i.e. intended for use or reference outside its own membership)
+			<em class=rfc2119>must</em> be published and promoted at a W3C-controlled URL,
+			and backed up by W3C systems
+			such that if the underlying service is discontinued,
+			W3C can continue to serve such content without breaking incoming links
+			or other key functionality.
+
+		<li>
+			All reports, publications, or other deliverables
+			produced by the group for public consumption
+			<em class=rfc2119>must</em> follow best practices for accessibility worldwide
+			and for persons with disabilities.
+			Network access to w3.org itself may be assumed.
```

Comment:
the rule above is on "W3C-controlled URL" which is distinct from w3.org (and not just in theory); it may be useful to align the phrasing and the intent.

Reply:
I think we were careful here. If a firewall or other device blocks access to w3.org, you can ignore that problem, is what we're saying. but if we re-direct, then you need to consider whether the target is reasonably reachable.

Reply:
W3C-controlled (as opposed to w3.org) was intentional. We want to allow other domains, provided systeam has the keys.

Reply:
I think the "systeam has the keys" may hide quite a bit of complexity; I know from experience that some of the non w3.org servers don't have the same level of reliability, and I don't think there is much the systeam can do about it, nor is it clear they would know how to replace the said servers should that be needed.

(this may be fixable, and the details of fixing it lies outside of the process; but if we want to endorse that type of approaches, I think there needs to be more operational details as to what they entail)

Reply:
Sure, but details like that don't go into the Process. :) The key point is that systeam can take over hosting if it the hosting service goes down permanently.

Reply:
Some recent events in IETF where "who owns the infrastructure" (beyond DNS) is having pretty consequential impact

Reply:
https://mailarchive.ietf.org/arch/msg/wgchairs/J3RxuqR6bhvlmoNGwfuilMrVjmU/

Reply:
The view expressed there seems fine by me. If W3C controls the domain, it doesn't matter who operates the service under that domain, they merely operate it, not own it.

As for the bigger point about assuming network access to $FOO, I actually agree with @dontcallmedom. The rule, as phrased, would imply that if a country blocks both w3.org and W3C-controlled foo-drafts.org, we'd have to move the stuff from foo-drafts.org to w3.org, even though it doesn't make anything better. So let's align the phrasing to “W3C-conrolled URL”.